cpp/src/IceBT/Engine.cpp

```diff
@@ -1256,7 +1256,7 @@ class BluetoothService : public DBus::Filter, public std::enable_shared_from_thi
         ConnectCallbackPtr _cb;
     };
 
-    IceUtil::Monitor<IceUtil::Mutex> _lock;
+    std::mutex _lock;
```

Comment:
Can we rename this to `_mutex`? 

Actually staring at the code, you're locking `_mutex` which used to be `_lock`. A bug? Am I missing something?